src/app/pages/login/login.component.html

```diff
@@ -1,8 +1,11 @@
 <div class="login-container">
-    <form name="form" (ngSubmit)="f.form.valid && login()" #f="ngForm" novalidate>
+    <div class ="msw-login-header">
+        <img src="/assets/img/school-logo.png" alt="" align="right">ביה"ס יחדיו
+      </div>
+        <form name="form" (ngSubmit)="f.form.valid && login()" #f="ngForm" novalidate>
           <div class="login-card">
             <div class="msw-login-logo-ph">
-              <img src="/assets/img/msw-proj-logo.png" alt="" width="80" height="80">
+              <img src="/assets/img/msw-proj-logo.png" alt="" width="80" height="80" style="background-color:white;">
```

Comment:
Can we move it to be in the SCSS file as a class?

Reply:
Done